watchos/PeopleInSpaceWatch/PeopleInSpaceWatch WatchKit Extension/ContentView.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  ContentView.swift
+//  PeopleInSpaceWatch WatchKit Extension
+//
+//  Created by Neal Sanche on 2020-01-02.
+//  Copyright © 2020 Neal Sanche. All rights reserved.
+//
+
+import SwiftUI
+import common
+
+struct ContentView: View {
+    
+    @ObservedObject var peopleInSpaceViewModel = PeopleInSpaceViewModel(repository: PeopleInSpaceRepository())
+    
+    var body: some View {
+        VStack {
+            List(peopleInSpaceViewModel.people, id: \.name) { person in
+                PersonView(person: person)
+            }
+            //.navigationBarTitle(Text("PeopleInSpace"), displayMode: .large)
```

Comment:
This needs a bit of work. It turns out if you use a NavigationLink inside PersonView below, you can get some master detail view navigation for free. I'll add that in for the example.